pkg/config/templates/rancherd-23-multus-config.yaml

```diff
@@ -6,7 +6,13 @@ resources:
     namespace: kube-system
   spec:
     valuesContent: |-
+      thickPlugin:
+        enabled: true
+      dynamicNetworksController:
+        enabled: true
+{{if eq . "arm64"}}
       labels:
-        nodeSelector: 
+        nodeSelector:
           kubernetes.io/os: linux
```

Comment:
the `kubernetes.io/os: linux` is also useful for x86 arch, as Harvester assumes to only run on Linux

could you also try to move L13 befor L17 ?